src/block/Anvil.php

```diff
@@ -108,7 +109,7 @@ public function onHitGround(FallingBlock $blockEntity) : void{
 					if($blockEntity->fallDistance > 1){
 						//If player has helmet do 1/4 the normal damage
 						$helmet = $ent->getArmorInventory()->getHelmet();
-						if($helmet !== null && !$helmet->isNull()){
+						if($helmet != null && !$helmet->isNull() && $helmet instanceof Durable){
```

Comment:
What happened here, reverted strict check?

Reply:
Because I got this PHPStan error.

`Error: Strict comparison using !== between pocketmine\item\Item and null will always evaluate to true.`

Reply:
The return value of `getHelmet()` is always Item.

Reply:
Yeah thats why I decided to just remove the check.